client/components/fraud-risk-tools-banner/index.tsx

```diff
@@ -50,11 +50,7 @@ const FRTDiscoverabilityBanner: React.FC = () => {
 		setDontShowAgain();
 	};
 
-	if ( ! showBanner ) {
-		return null;
-	}
-
-	return (
+	return showBanner ? (
```

Comment:
_Nitpick_: For cases like this, we tend to use the early return syntax to avoid nesting the JSX code.

Anyway, just a matter of taste 🙂